Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ImageIO"
 uuid = "82e4d734-157c-48bb-816b-45c225c6df19"
 authors = ["Ian Butterworth"]
-version = "0.6.7"
```

Comment:
Not strictly related to this PR, but the repo is missing a tag/release for 0.6.7 it seems...

Reply:
Fixed, but TagBot isn't happy even though I updated it to latest config and refreshed the DOCUMENTER_KEY etc.